src/query/service/src/interpreters/access/privilege_access.rs

```diff
@@ -232,6 +232,18 @@ impl AccessChecker for PrivilegeAccess {
                     )
                     .await?;
             }
+            Plan::SetOptions(plan) => {
+                session
+                    .validate_privilege(
+                        &GrantObject::Table(
+                            plan.catalog.clone(),
+                            plan.database.clone(),
+                            plan.table.clone(),
+                        ),
+                        vec![UserPrivilegeType::Alter],
```

Comment:
`UserPrivilegeType::Super` would be better, since altering arbitrary table options is risky, if this will not bring too much trouble to unit / integration testings

Reply:
sure!!

Reply:
turns out, using `Super` privilege does bring troubles to ut/it : ( let's address it later

Reply:
ok